Felienne/hedy
# Add types to SEMANTICS description for first levels

**Description**

Adds types per command in the semantics description for level 1 and 2 

**How to test**

Read SEMANTICS.md file. Unsure about the types where I put a questionmark behind it, so maybe check those, and see if lay-out is okay like this.
